R/pkg/R/functions.R

```diff
@@ -361,10 +361,13 @@ setMethod("column",
 #'
 #' @rdname corr
 #' @name corr
-#' @family math functions
+#' @family aggregate functions
 #' @export
 #' @aliases corr,Column-method
-#' @examples \dontrun{corr(df$c, df$d)}
+#' @examples
+#' \dontrun{
+#' df <- createDataFrame(cbind(model = rownames(mtcars), mtcars))
```

Comment:
or maybe we should have a newline at the end of every `@example` block (when there are multiple examples on one Rd)? This way we don't have to know where goes first